### PR TITLE
ansible-lint: fix failing syntax checks

### PR DESCRIPTION
**Pull request by guits**
[skip ci]
Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

**Comment by guits** [comment]
jenkins test playbook syntax

**Comment by guits** [comment]
jenkins test playbook syntax

**Comment by guits** [comment]
jenkins test playbook syntax

**Comment by guits** [comment]
jenkins test playbook syntax

File: roles/ceph-client/tasks/create_users_keys.yml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   when: not containerized_deployment
 
 - name: create key(s)
-  shell: "{{ docker_exec_client_cmd }} -C /etc/ceph/{{ cluster }}.{{ item.name }}.keyring --name {{ item.name }} --add-key {{ item.key }} --cap mon \"{{ item.mon_cap|default('') }}\" --cap osd \"{{ item.osd_cap|default('') }}\" --cap mds \"{{ item.mds_cap|default('') }}\""
+  command: "{{ docker_exec_client_cmd }} -C /etc/ceph/{{ cluster }}.{{ item.name }}.keyring --name {{ item.name }} --add-key {{ item.key }} --cap mon \"{{ item.mon_cap|default('') }}\" --cap osd \"{{ item.osd_cap|default('') }}\" --cap mds \"{{ item.mds_cap|default('') }}\""
```
Comment (leseb) [inline roles/ceph-client/tasks/create_users_keys.yml]: looks like shell is still need here

**Pull request closed without merging**